src/components/date/date.js

```diff
@@ -129,7 +137,16 @@ const Date = Input(InputIcon(InputLabel(InputValidation(class Date extends React
     * @type {Array}
     * @default DateValidator
     */
-    internalValidations: [new DateValidator()]
+    internalValidations: [new DateValidator()],
+
+    /**
+     * Display the date of today if the value is empty
+     *
+     * @property showTodayIfEmpty
+     * @type {Boolean}
+     * @default true
+     */
+    showTodayIfEmpty: true
```

Comment:
Whilst this work I think it's unnecessary to add the prop in this way, you could have a boolean prop called `allowEmptyValue` that would allow the date input to display an empty string. Doing this maintains backwards compatibility and without having to set a default prop to do so.